wdl/qc_imputation.wdl

```diff
@@ -375,14 +783,14 @@ task batch_qc {
         $plink2_cmd --bfile ${base} --remove <(awk '{OFS="\t"; print "0",$1}' ${base}.samples_exclude.txt | sort -u) \
               --hardy --hwe ${hw} --make-bed --out plink_data_after_hwe
         awk ' NR==1{ for(i=1;i<=NF;i++)  { h[$i]=i; if((!"ID" in h) || (! "P" in h)) { print "ID and P fields expected in HARDY" >>"/dev/stderr"; exit 1} } }  \
-            NR>1&&$h["P"]<${hw} {OFS="\t"; print $h["ID"],"hwe",$h["P"]}' plink_data_after_hwe.hardy >> ${base}.variants_exclude.txt
+            NR>1&&$h["P"]<${hw} {OFS="\t"; print $h["ID"],"hwe",$h["P"]}' plink_data_after_hwe.hardy >> ${base}.exclude_variants_batch_qc.txt
         cp plink_data_after_hwe.hardy ${base}.hardy
 
         awk '$4==2{ print $1,$2}' ${base}.sexcheck > genetic_females
         $plink_cmd --remove <(awk '{OFS="\t"; print "0",$1}' ${base}.samples_exclude.txt | sort -u) --bfile ${base} \
             --chr 23 --keep genetic_females --hardy --out females_only
         awk 'NR==1{ for(i=1;i<=NF;i++)  { h[$i]=i; if((!"SNP" in h) || (! "P" in h)) { print "ID and P fields expected in HARDY" >>"/dev/stderr"; exit 1} } } \
-            NR>1&&$h["P"]<${hw} {OFS="\t"; print $h["SNP"],"hwe",$h["P"]}' females_only.hwe >> ${base}.variants_exclude.txt
+            NR>1&&$h["P"]<${hw} {OFS="\t"; print $h["SNP"],"hwe",$h["P"]}' females_only.hwe >> ${base}.exclude_variants_batch_qc.txt
```

Comment:
we should also here ignore hwe in case of deficiency and low af as in the joint qc.

Reply:
Agreed - made an issue

Reply:
grreat!